apps/explorer/views.py

```diff
@@ -68,3 +72,39 @@ def get_queryset(self):
             'pixels__omics_unit__type',
             'pixels__omics_unit__strain__species',
         )
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context.update({
+            'export_form': PixelSetExportForm(),
+        })
+        return context
+
+
+class PixelSetExportView(LoginRequiredMixin, FormView):
+    ATTACHEMENT_FILENAME = 'pixelsets.zip'
```

Comment:
Why not making the archive name almost unique by adding time related information?

```python
from django.utils import timezone

ATTACHEMENT_FILENAME = 'pixelsets_{now}.zip'

# This can be an util or a method of your view
def get_export_archive_filename():
    return ATTACHEMENT_FILENAME.format(
        now=timezone.now().strftime('%Y%m%d_%Hh%Mm%Ss%f')
    )

# We expect something like
# pixelsets_20180126_09h18m21s609124.zip
```

You can mock `timezone.now` for your tests.